apps/playground/src/stories/hooks/useHover.stories.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useCallback, useMemo, useRef, useState } from 'react';
+import { useHover } from '../../../../../packages/react-hooks/use-hover';
+// import { useHover } from "@nanlabs/react-hooks";
+
+export const Example = () => {
+  const [isCbMemoBroken, setIsCbMemoBroken] = useState(false);
```

Comment:
@boredcity could you use `isMemoEnabled` instead of `isCbMemoBroken`?

Reply:
I ask for this change because for this case the name `broken` is not self-explanatory enough.

Reply:
Agreed